scripts/select_appointments.sql

```diff
@@ -1 +1,19 @@
 -- retrieve appointment information
+
+SELECT d.family_name AS doctor, p.family_name, p.given_name, a.start_at, a.length
```

Comment:
This has a similar problem to the script where you are trying to insert rows where you're referencing two columns, `start_at` and `length`, that don't exist